articles/compliance/gdpr/features-aiding-compliance/user-consent.md

```diff
@@ -0,0 +1,353 @@
+
+```har
+{
+    "method": "PATCH",
+    "url": "https://${account.namespace}/api/v2/users/USER_ID",
+    "httpVersion": "HTTP/1.1",
+    "headers": [{
+        "name": "Authorization",
+        "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+    },
+    {
+        "name": "content-type",
+        "value": "application/json"
+    }],
+    "postData" : {
+        "mimeType": "application/json",
+        "text": "{\"user_metadata\":{\"consentDate\":\"01/24/2018\"}}"
+    }
+}
+```
+
+This will add a new property to the user profile, the **user_metadata.consentDate**, which will hold the date the customer consented. The response will be the full user profile. The updated metadata will look like this:
+
+```json
+{
+  "consentGiven": true,
+  "consentDate": "01/23/2018",
+  "consentDetails": "some-url"
+}
+```
+
+#### Update an inner property
+
+In this case you have to send the whole object, even though you might want to update only one property. If you do not, the other properies will be removed. Let's add an inner property for the consent date and set it to `01/23/2018`.
+
+```har
+{
+    "method": "PATCH",
+    "url": "https://${account.namespace}/api/v2/users/USER_ID",
+    "httpVersion": "HTTP/1.1",
+    "headers": [{
+        "name": "Authorization",
+        "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+    },
+    {
+        "name": "content-type",
+        "value": "application/json"
+    }],
+    "postData" : {
+        "mimeType": "application/json",
+        "text": "{\"user_metadata\":{\"consent\": {\"given\":true, \"date\":\"01/23/2018\", \"text_details\":\"some-url\"}}}"
+    }
+}
+```
+
+This will add a new property to the user profile, the **user_metadata.consent.date**, which will hold the date the customer consented. The response will be the full user profile. The updated metadata will look like this:
+
+```json
+{
+  "consent": {
+    "given": true,
+    "date": "01/23/2018",
+    "text_details": "some-url"
+  }
+}
+```
+
+### Export consent information
+
+To export a list of your users using the Management API, use [the User export endpoint](/users/search#user-export). 
+
+This endpoint creates a job that exports all users associated with a connection. You will need the Id of the connection. To find this Id, use [the Get Connections endpoint](/api/management/v2#!/Connections/get_connections) (you can set the **name** parameter to the name of the connection to retrieve only this one).
+
+Once you have the connection Id and a [Management API token](/api/management/v2/tokens), you are ready to start exporting users. For a sample request and response see [User Export](/users/search#user-export).
+
+---
+
+:::panel What else do I have to do?
+- Determine how you want to track consent. We recommend putting in not just the date but the correct version of terms and conditions, specifying the version, and including an array for those who will withdraw, provide and withdraw consent again.
```

Comment:
Determine how you want to track user consent. We recommend including information on not just the date the user consented, but the version of terms and conditions to which the user agreed. We also recommend including an array to hold information about users that withdraw their permission -- remember that the user can consent and withdraw multiple times.